arviz/plots/elpdplot.py

```diff
@@ -5,7 +5,7 @@
 from ..data import convert_to_inference_data
 from ..rcparams import rcParams
 from ..stats import ELPDData, loo, waic
-from ..utils import get_coords
+from ..utils import get_coords, Calculate_ICS
```

Comment:
As this is a function, it should follow the `snake_case` naming convention, all lowercase and underscores instead of spaces, dashes or capitalization.